index.html

```diff
@@ -0,0 +1,116 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title> Build A Survey Form</title>
+        <!-- Added css file  -->
+        <link rel="stylesheet" href="style.css">
+    </head>
+    <body>
+        <div class="container">
+
+            <header class="header">
+                <h1 id="title" class="text-center">Microvarse Survey Form</h1>
+                <p id="description" class="description text-center">
+                    Thanks for taking the time to help us improve the platform
+                </p>
+            </header>
```

Comment:
You have given reasons why the user should make the required change but I think we can also improve this comment by being more suggestive rather than instructive or giving orders😉😉 kindly refer back to the first comment for a clarity🤗